### PR TITLE
BZs node-1

### PR DESCRIPTION
**Pull request by jeana-redhat**
BZs for Node, batch 1

preview: https://deploy-preview-36998--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes.html#ocp-4-9-bug-fixes

### CODE REVIEW

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -935,6 +935,18 @@ The deprecated `dhclient` binary has been removed from {op-system}. Starting wit
 
 *Node*
 
+* Previously, in containers, CRI-O did not create a symbolic link from `/proc/mounts` file to the `/etc/mtab` file. As a consequence, the user could not view the list of the mounted devices in the container's `/etc/mtab` file. CRI-O now adds the symbolic link. As a result, users can view the container's mounted devices.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1868221[*BZ#1868221*])
+
+* Previously, if pods were deleted quickly after creation, the kubelet might not clean up the pods properly. This resulted in pods being stuck in terminating state could impact availability of upgrades. This fix improves the pod lifecycle logic to avoid this problem.
```
Comment (stevsmit) [inline release_notes/ocp-4-9-release-notes.adoc:941]: Does the second sentence read weird to you?

File: release_notes/ocp-4-9-release-notes.adoc [hunk 2]
```diff
@@ -935,6 +935,18 @@ The deprecated `dhclient` binary has been removed from {op-system}. Starting wit
...
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1952224[*BZ#1952224*])
+
+* Previously, the `SystemMemoryExceedsReserved` alert would fire when the system memory usage exceeded 90% of the reserved memory. As a result, clusters could fire an excessive number of alerts. The threshold for this alarm was changed to fire at 95% of reserved memory.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1980844[*BZ#1980844*])
+
+* Previously, a bug in CRI-O caused CRI-O to leak a child PID of a process it created. As a result, if under load, Systemd could create a significant number of zombie processes. This could lead to node failure if the node ran out of PIDs. CRI-O was fixed to prevent the leakage. As a result, these zombie process are no longer being created.
```
Comment (stevsmit) [inline release_notes/ocp-4-9-release-notes.adoc:947]: systemd

Comment (bobfuru) [reply release_notes/ocp-4-9-release-notes.adoc:947]: also, should be "these zombie processes" (I relate to this on a meta-level today)

File: release_notes/ocp-4-9-release-notes.adoc (hunk 1 above)
Comment (jeana-redhat) [reply release_notes/ocp-4-9-release-notes.adoc:941]: yes, will see about fixing that

File: release_notes/ocp-4-9-release-notes.adoc (hunk 2 above)
Comment (jeana-redhat) [reply release_notes/ocp-4-9-release-notes.adoc:947]: #mood

**Comment by jeana-redhat** [comment]
For https://issues.redhat.com/browse/OSDOCS-2514

**Pull request merged**